e2e/testdata/simple-bundle.json.golden

```diff
@@ -106,6 +106,16 @@
 				"com.docker.app.render"
 			]
 		},
+		"docker.share-registry-creds": {
```

Comment:
self review: this (and possible some other  things) should be `com.docker.app.foo` now I think,